elintegrostartapp/grails-app/services/elintegroWebsite/ContactUsMessageService.groovy

```diff
@@ -0,0 +1,26 @@
+package elintegroWebsite
+
```

Comment:
Let us move it to the package com.elintegro.website (under the "service" folder)